lib/companies/schema/company.ex

```diff
@@ -1,27 +1,33 @@
 defmodule Companies.Schema.Company do
   @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
```

Comment:
❤️ 

I briefly thought about adding ecto and using a changeset to do the validations in #711, but I wasn't sure about the overhead of adding ecto when it wasn't used in tandem with a DB. Glad you went the way of ecto.

Reply:
We don't need to add a DB adapter to Ecto and we can specify it's dev/test only to eliminate the production overhead.